ee_tests/cico_run_e2e_tests.sh

```diff
@@ -162,11 +162,15 @@ if [ "$ZABBIX_ENABLED" = true ] ; then
 fi
 
 if [[ "$TEST_SUITE" = "che" ]]; then
+    mkdir "$ARTIFACTS_DIR"/che
+    mkdir "$ARTIFACTS_DIR"/che/failsafe
```

Comment:
It's enough to call the second command with -p param, i.e. `mkdir -p "$ARTIFACTS_DIR"/che/failsafe`

Reply:
Done.